src/frontend/src/pages/GanttPage/GanttChartComponents/GanttTaskBar.tsx

```diff
@@ -94,6 +107,18 @@ const GanttTaskBar = ({
     createChange({ id, eventId: event.id, type: 'shift-by-days', days });
   };
 
+  const handleOnMouseOver = (e: React.MouseEvent) => {
+    if (!isEditMode) {
+      setCursorX(e.clientX);
+      setCursorY(e.clientY);
+      setShowPopup(true);
```

Comment:
for the popup, can we make it a red header with a grey body (that dark grey we use on pageblock backgrounds, make sure to use useTheme so this looks good in light mode as well), just to make it look a little nicer. Also, let's try just putting the title in the header, will hopefully look cleaner and they can already see the wbsNum on the gantt chart
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/2801fb58-7bc8-4192-80a9-1245970dc4c9)